corehq/apps/zapier/api/v0_5.py

```diff
@@ -40,6 +55,21 @@ def _has_default_label(self, question):
         return question['label'] == question['hashtagValue']
 
     def obj_get_list(self, bundle, **kwargs):
+        raise NotImplementedError
+
+    class Meta(CustomResourceMeta):
+        object_class = CustomField
+        resource_name = 'custom_fields_base'
```

Comment:
does this need a `resource_name`? as the base class, I don't think this should be a real resource, since all the others will override this property.